src/app/flows/fines/fines-mac/fines-mac-account-details/fines-mac-account-details.component.html

```diff
@@ -317,7 +317,7 @@ <h2 class="govuk-heading-m">Review and publish</h2>
 </ng-template>
 
 <ng-template #paymentTermsStatus>
-  @if (accountCreationStatus['personalDetails'] || defendantType === defendantTypes['company']) {
+  @if (accountCreationStatus['personalDetails'] || paymentTermsBypassDefendantTypes.includes(defendantType)) {
```

Comment:
Would it be better to wrap this logic in a function in the *.component.ts?

```
isCreationStatusOrWhatever(){
    return accountCreationStatus['personalDetails'] || paymentTermsBypassDefendantTypes.includes(defendantType)
}
```

I think this makes the template a bit more readable and the logic more testable